src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/table.tsx

```diff
@@ -0,0 +1,272 @@
+              ),
+            }
+          )}
+          size="xs"
+          position="top"
+          isHoverable
+        />
+      </Title>
+      <StyledPanelTable
+        headers={[
+          t('Status'),
+          t('Debug File'),
+          t('Processing'),
+          t('Features'),
+          t('Actions'),
+        ]}
+        isEmpty={!candidates.length}
+        isLoading={isLoading}
+      >
+        {candidates.map((candidate, index) => {
+          const {location, download, source_name, source} = candidate;
+          const isInternalSource = source === INTERNAL_SOURCE;
+          return (
+            <React.Fragment key={index}>
+              <StatusColumn>
+                <Status candidate={candidate} />
+              </StatusColumn>
+
+              <DebugFileColumn>
+                <Tooltip title={getSourceTooltipDescription(source)}>
+                  <SourceName>{source_name ?? t('Unknown')}</SourceName>
+                </Tooltip>
+                {location && !isInternalSource && (
+                  <ClipboardTooltip title={location} onSuccess={() => onCopy(location)}>
+                    <Location>{location}</Location>
+                  </ClipboardTooltip>
+                )}
+              </DebugFileColumn>
+
+              <ProcessingColumn>{renderProcessing(download)}</ProcessingColumn>
+
+              <FeaturesColumn>{renderFeatures(download)}</FeaturesColumn>
+
+              <ActionsColumn>
+                {isInternalSource && (
+                  <Actions
+                    onDelete={onDelete}
+                    baseUrl={baseUrl}
+                    projectId={projectId}
+                    organization={organization}
+                    candidate={candidate}
+                  />
+                )}
+              </ActionsColumn>
+            </React.Fragment>
+          );
+        })}
+      </StyledPanelTable>
+    </Wrapper>
+  );
+}
+
+export default Table;
+
+Table.propTypes = {
+  organization: SentryTypes.Organization,
+};
+
+const Wrapper = styled('div')`
+  display: grid;
+  grid-gap: ${space(1)};
+`;
+
+const StyledPanelTable = styled(PanelTable)`
+  grid-template-columns: max-content minmax(185px, 1.5fr) 1fr 1fr;
+
+  > *:nth-child(5n) {
+    padding: 0;
+    display: none;
+  }
+
+  > *:nth-child(5n-1),
+  > *:nth-child(5n) {
+    text-align: right;
+    justify-content: flex-end;
+  }
+
+  @media (min-width: ${p => p.theme.breakpoints[2]}) {
+    overflow: visible;
+
+    > *:nth-child(5n-1) {
+      text-align: left;
+      justify-content: flex-start;
+    }
+
+    > *:nth-child(5n) {
+      padding: ${space(2)};
+      display: flex;
+    }
+
+    grid-template-columns: max-content minmax(185px, 1.5fr) 1fr 1fr max-content;
+  }
+`;
+
+// Table Title
+const Title = styled('div')`
+  display: grid;
+  grid-gap: ${space(0.5)};
+  grid-template-columns: repeat(2, max-content);
+  align-items: center;
+  font-weight: 600;
+  color: ${p => p.theme.gray400};
```

Comment:
Does this look ok in dark mode?

Reply:
yes, imo it looks fine 
![image](https://user-images.githubusercontent.com/29228205/104004161-b69a0b80-51a3-11eb-879f-e58336034079.png)